### PR TITLE
first batch of Readme updates

### PR DESCRIPTION
**Pull request by xinity**
README update WIP.

leave the PR open until all updates are committed.
comments welcome of course :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,7 +138,8 @@ There are 5 ways to install Gogs:
 
 ## Contributors
 
-- Ex-team members [@lunny](https://github.com/lunny), [@fuxiaohei](https://github.com/fuxiaohei) and [@slene](https://github.com/slene).
+- Owners [@lunny](https://github.com/lunny), [@tboerger](https://github.com/tboerger) and [@bkcsoft](https://github.com/bkcsoft).
```
Comment (tboerger) [inline README.md]: I would not mention that here, we can skip the maintainance of this part. You can see that anyway on the people page of the org.

File: README.md [hunk 2]
```diff
@@ -19,7 +20,7 @@
 
 ### Important Notes
 
-1. **YOU MUST READ [Contributing Code](https://github.com/go-gitea/gitea/wiki/Contributing-Code) BEFORE STARTING TO WORK ON A PULL REQUEST**.
+1. **YOU MUST READ [Contributing Code](https://github.com/go-gitea/gitea/CONTRIBUTING.md) BEFORE STARTING TO WORK ON A PULL REQUEST**.
```
Comment (tboerger) [inline README.md]: `You must read the [contributors guide]() before starting to work on a pull request`

File: README.md [hunk 3]
```diff
@@ -7,9 +7,10 @@
 [![](https://images.microbadger.com/badges/image/gitea/gitea.svg)](http://microbadger.com/images/gitea/gitea "Get your own image badge on microbadger.com")
 [![Join the chat at https://gitter.im/go-gitea/gitea](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/go-gitea/gitea?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-![](https://github.com/go-gitea/gitea/blob/master/public/img/gogs-large-resize.png?raw=true)
+![](https://github.com/go-gitea/gitea/blob/master/public/img/gitea-large-resize.png?raw=true)
 
-##### Current tip version: 0.9.99 (see [Releases](https://github.com/go-gitea/gitea/releases) for binary versions or submit a task on [alpha stage automated binary building system](https://build.gogs.io/))
+##### Status
+## Current tip version: 0.9.99 (see [Releases](https://github.com/go-gitea/gitea/releases)
```
Comment (tboerger) [inline README.md]: This should not be a headline, beside that this branch will be 1.0.0

**Review by tboerger — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (xinity) [reply README.md]: agree @lunny 
i though at first to remove this line and simply link to the release page.

**Comment by xinity** [comment]
I've taken care of changes requested, but github stills ask for them to be taken care of.
Did i miss something ?

**Comment by strk** [comment]
I can see a "Dismiss review" link next to the "requested changes" record. I guess all maintainers do see that.
But given @tboerger requested the change I guess he's the one who should click that link, to confirm his request was addressed

File: README.md [hunk 4]
```diff
@@ -7,9 +7,10 @@
 [![](https://images.microbadger.com/badges/image/gitea/gitea.svg)](http://microbadger.com/images/gitea/gitea "Get your own image badge on microbadger.com")
 [![Join the chat at https://gitter.im/go-gitea/gitea](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/go-gitea/gitea?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-![](https://github.com/go-gitea/gitea/blob/master/public/img/gogs-large-resize.png?raw=true)
+![](https://github.com/go-gitea/gitea/blob/master/public/img/gitea-large-resize.png?raw=true)
 
-##### Current tip version: 0.9.99 (see [Releases](https://github.com/go-gitea/gitea/releases) for binary versions or submit a task on [alpha stage automated binary building system](https://build.gogs.io/))
+##### Status
+**Current version: 1.0** (see [Releases](https://github.com/go-gitea/gitea/releases))
```
Comment (strk) [inline README.md]: I think versions have 3 numbers, so would be 1.0.0 (but there's no such release made yet).
I agree on dropping this paragraph completely and just link to the releases page to figure out what the "current version" is.

File: README.md [hunk 5]
```diff
@@ -19,10 +20,10 @@
 
 ### Important Notes
 
-1. **YOU MUST READ [Contributing Code](https://github.com/go-gitea/gitea/wiki/Contributing-Code) BEFORE STARTING TO WORK ON A PULL REQUEST**.
+1. **YOU MUST READ THE [Contributors Code](https://github.com/go-gitea/gitea/CONTRIBUTING.md) BEFORE STARTING TO WORK ON A PULL REQUEST**.
```
Comment (strk) [inline README.md]: @tboerger asked to use "contributors guide" here, IIRC

File: README.md [hunk 6]
```diff
@@ -19,10 +20,10 @@
...
 2. Due to testing purpose, data of [try.gogs.io](https://try.gogs.io) was reset in **Jan 28, 2015** and will reset multiple times after. Please do **NOT** put your important data on the site.
 3. The demo site [try.gogs.io](https://try.gogs.io) is running under `develop` branch.
-4. If you think there are vulnerabilities in the project, please talk privately to **u@gogs.io**. Thanks!
+4. If you think there are vulnerabilities in the project, please talk privately to **lunny@gitea.io**. Thanks!
```
Comment (tboerger) [inline README.md]: security@gitea.io

Comment (strk) [inline README.md]: @lunny wasn't there a wider address to contact owners/maintainers ?

File: README.md (hunk 4 above)
Comment (tboerger) [inline README.md]: Please add a blank line here like on every headline

File: README.md [hunk 7]
```diff
@@ -71,15 +72,17 @@ The goal of this project is to make the easiest, fastest, and most painless way
 
 ## Installation
 
+**Note: As Gitea is a [gogs](https://github.com/gogits/gogs) fork, tutorials and documentation related to gogs applies to Gitea too**
```
Comment (tboerger) [inline README.md]: Gogs

File: README.md [hunk 8]
```diff
@@ -71,15 +72,17 @@ The goal of this project is to make the easiest, fastest, and most painless way
...
+
 Make sure you install the [prerequisites](https://gogs.io/docs/installation) first.
 
-There are 5 ways to install Gogs:
+There are 5 ways to install Gitea:
 
 - [Install from binary](https://gogs.io/docs/installation/install_from_binary.html)
 - [Install from source](https://gogs.io/docs/installation/install_from_source.html)
 - [Install from packages](https://gogs.io/docs/installation/install_from_packages.html)
```
Comment (strk) [inline README.md]: I would drop the gogs.io links, those would not be releases of Gitea anyway.
Maybe also drop the count of ways to install, as that number is going to change a few times before becoming stable :)

**Review by strk — CHANGES_REQUESTED** [review]

File: README.md (hunk 8 above)
Comment (xinity) [reply README.md]: i agree but then there won't be any installation documentation links until we write one.
Doesn't matter ?

File: README.md [hunk 9]
```diff
@@ -7,9 +7,11 @@
 [![](https://images.microbadger.com/badges/image/gitea/gitea.svg)](http://microbadger.com/images/gitea/gitea "Get your own image badge on microbadger.com")
 [![Join the chat at https://gitter.im/go-gitea/gitea](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/go-gitea/gitea?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-![](https://github.com/go-gitea/gitea/blob/master/public/img/gogs-large-resize.png?raw=true)
+![](https://github.com/go-gitea/gitea/blob/master/public/img/gitea-large-resize.png?raw=true)
 
-##### Current tip version: 0.9.99 (see [Releases](https://github.com/go-gitea/gitea/releases) for binary versions or submit a task on [alpha stage automated binary building system](https://build.gogs.io/))
+##### Status
+
+**Current version: 1.0.0** (see [Releases](https://github.com/go-gitea/gitea/releases))
```
Comment (strk) [inline README.md]: How about drop the version number (will easily get outdated) and just point to the releases page ?

File: README.md [hunk 10]
```diff
@@ -19,25 +21,14 @@
...
 
-1. **YOU MUST READ [Contributing Code](https://github.com/go-gitea/gitea/wiki/Contributing-Code) BEFORE STARTING TO WORK ON A PULL REQUEST**.
-2. Due to testing purpose, data of [try.gogs.io](https://try.gogs.io) was reset in **Jan 28, 2015** and will reset multiple times after. Please do **NOT** put your important data on the site.
-3. The demo site [try.gogs.io](https://try.gogs.io) is running under `develop` branch.
-4. If you think there are vulnerabilities in the project, please talk privately to **u@gogs.io**. Thanks!
-5. If you're interested in using APIs, we have experimental support with [documentation](https://godoc.org/github.com/go-gitea/go-sdk).
-6. If your team/company is using Gogs and would like to put your logo on [our website](https://gogs.io), contact us by any means.
+1. **YOU MUST READ THE [Contributors Guide](https://github.com/go-gitea/gitea/CONTRIBUTING.md) BEFORE STARTING TO WORK ON A PULL REQUEST**.
+4. If you think there are vulnerabilities in the project, please talk privately to **security@gitea.io**. Thanks!
+6. If your team/company is using Gogs and would like to put your logo on [our website](https://gitea.io), contact us by any means.
```
Comment (strk) [inline README.md]: is using *Gitea ?

File: README.md [hunk 11]
```diff
@@ -69,15 +60,14 @@ The goal of this project is to make the easiest, fastest, and most painless way
 
 ## Installation
 
-Make sure you install the [prerequisites](https://gogs.io/docs/installation) first.
+**Note: As Gitea is a [Gogs](https://github.com/gogits/gogs) fork, tutorials and documentation related to gogs applies to Gitea too**
 
-There are 5 ways to install Gogs:
+There are 2 ways to install Gitea:
```
Comment (strk) [inline README.md]: Drop the count, those ways are going to change and we'll forget to update the count :)

File: README.md [hunk 12]
```diff
@@ -133,7 +123,7 @@ There are 5 ways to install Gogs:
 
 ## Contributors
 
-- Ex-team members [@lunny](https://github.com/lunny), [@fuxiaohei](https://github.com/fuxiaohei) and [@slene](https://github.com/slene).
+- see [Maintainer page](https://github.com/orgs/go-gitea/people)
```
Comment (strk) [inline README.md]: Capitalize See ?

**Comment by strk** [comment]
Reviewed, feel free to ignore my review and have my LGTM :)

File: README.md (hunk 12 above)
Comment (xinity) [reply README.md]: not sure about this one

File: README.md (hunk 11 above)
Comment (xinity) [reply README.md]: done will push

File: README.md (hunk 10 above)
Comment (xinity) [reply README.md]: done will push

File: README.md (hunk 9 above)
Comment (xinity) [reply README.md]: done will push

File: README.md [hunk 13]
```diff
@@ -19,25 +21,14 @@
 
 ### Important Notes
 
-1. **YOU MUST READ [Contributing Code](https://github.com/go-gitea/gitea/wiki/Contributing-Code) BEFORE STARTING TO WORK ON A PULL REQUEST**.
-2. Due to testing purpose, data of [try.gogs.io](https://try.gogs.io) was reset in **Jan 28, 2015** and will reset multiple times after. Please do **NOT** put your important data on the site.
-3. The demo site [try.gogs.io](https://try.gogs.io) is running under `develop` branch.
-4. If you think there are vulnerabilities in the project, please talk privately to **u@gogs.io**. Thanks!
-5. If you're interested in using APIs, we have experimental support with [documentation](https://godoc.org/github.com/go-gitea/go-sdk).
```
Comment (tboerger) [inline README.md:26]: I would keep that and link to the godocs.

File: README.md [hunk 14]
```diff
@@ -19,25 +21,14 @@
...
-6. If your team/company is using Gogs and would like to put your logo on [our website](https://gogs.io), contact us by any means.
+1. **YOU MUST READ THE [Contributors Guide](https://github.com/go-gitea/gitea/CONTRIBUTING.md) BEFORE STARTING TO WORK ON A PULL REQUEST**.
+4. If you think there are vulnerabilities in the project, please talk privately to **security@gitea.io**. Thanks!
```
Comment (tboerger) [inline README.md]: Numbering?

File: README.md [hunk 15]
```diff
@@ -19,25 +21,14 @@
...
+6. If your team/company is using Gitea and would like to put your logo on [our website](https://gitea.io), contact us by any means.
```
Comment (tboerger) [inline README.md]: I would drop that for now.

**Review by tboerger — CHANGES_REQUESTED** [review]

File: README.md [hunk 16]
```diff
@@ -133,7 +123,7 @@ There are 5 ways to install Gogs:
 
 ## Contributors
 
-- Ex-team members [@lunny](https://github.com/lunny), [@fuxiaohei](https://github.com/fuxiaohei) and [@slene](https://github.com/slene).
+- See [Maintainer page](https://github.com/orgs/go-gitea/people)
 - See [contributors page](https://github.com/go-gitea/gitea/graphs/contributors) for full list of contributors.
```
Comment (ghost) [inline README.md]: Possibly `[Contributors]`, `[Maintainer]` and `[Translators]`?

File: README.md (hunk 15 above)
Comment (ghost) [reply README.md]: Agreed, we just became open source after all.

**Review by strk — APPROVED** [review]

File: README.md (hunk 14 above)
Comment (xinity) [reply README.md]: done will push

File: README.md (hunk 15 above)
Comment (xinity) [reply README.md]: done will push

File: README.md (hunk 13 above)
Comment (xinity) [reply README.md:26]: done will push

File: README.md [hunk 17]
```diff
@@ -22,8 +22,8 @@
 ### Important Notes
 
 1. **YOU MUST READ THE [Contributors Guide](https://github.com/go-gitea/gitea/CONTRIBUTING.md) BEFORE STARTING TO WORK ON A PULL REQUEST**.
-4. If you think there are vulnerabilities in the project, please talk privately to **security@gitea.io**. Thanks!
-6. If your team/company is using Gitea and would like to put your logo on [our website](https://gitea.io), contact us by any means.
+2. If you think there are vulnerabilities in the project, please talk privately to **security@gitea.io**. Thanks!
+3. If you're interested in using APIs, we have experimental support with [documentation](https://godoc.org/github.com/go-gitea/go-sdk).
```
Comment (strk) [inline README.md:26]: Gogs came back ?

**Comment by xinity** [comment]
@strk are you sure?

File: README.md (hunk 17 above)
Comment (strk) [reply README.md:26]: sorry, I was wrong, all looks fine

**Comment by lunny** [comment]
seems it's ready for merge. LGTM

File: README.md [hunk 18]
```diff
@@ -69,15 +60,14 @@ The goal of this project is to make the easiest, fastest, and most painless way
 
 ## Installation
 
-Make sure you install the [prerequisites](https://gogs.io/docs/installation) first.
+**Note: As Gitea is a [Gogs](https://github.com/gogits/gogs) fork, tutorials and documentation related to gogs applies to Gitea too**
```
Comment (bkcsoft) [inline README.md:63]: `fork of [Gogs]` sounds better no?

**Comment by strk** [comment]
On Wed, Nov 09, 2016 at 06:30:13AM -0800, bkcsoft wrote:

> `fork of [Gogs]` sounds better no?

Nitpicking

**Comment by bkcsoft** [comment]
@strk which is why I left it as a comment instead of `required change` :wink:

**Comment by strk** [comment]
Do do we have your LGTM @bkcsoft ? So we can merge and move on

**Comment by lunny** [comment]
@tboerger we need your approval changes.

**Review by tboerger — APPROVED** [review]

**Pull request merged**